packages/react-native-reanimated/src/ReanimatedModule/NativeReanimated.ts

```diff
@@ -13,11 +14,15 @@ import { getValueUnpackerCode } from '../valueUnpacker';
 import { isFabric } from '../PlatformChecker';
 import type React from 'react';
 import { getShadowNodeWrapperFromRef } from '../fabricUtils';
-import ReanimatedModule from '../specs/NativeReanimatedModule';
+import { ReanimatedTurboModule } from '../specs';
```

Comment:
At some point, I'd like to make a clear distinction between `ReanimatedModule` and `ReanimatedTurboModule`. The sole purpose of the latter is to install JSI bindings so let's rename it to `ReanimatedInstallerTurboModule`.

Reply:
I'd rather stay with `ReanimatedTurboModule`. `ReanimatedTurboModule` is indeed a JavaScript TurboModule instance - it's just that we only use it to call its' `install` function and handle JSI calls with `NativeReanimatedModule`.